lib/utils/UIConstants.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:ui';
+
+abstract class UIConstants {
```

Comment:
keep the naming convention of the files the same, lower case separated with underscores